sci-misc/boinc/metadata.xml

```diff
@@ -19,7 +19,8 @@
   Use nvidia cuda toolkit for speeding up computations.
   NOTE: works only for subset of nvidia graphic cards so make sure your card
   is supported before opening a bug about it.
-  </flag>
+    </flag>
+    <flag name="opencl">Enable OpenCL support</flag>
```

Comment:
Uh-Oh... Then I will remove the flag, and instead add a post install text that points out that OpenCL is supported, and that it is up to the user to install it.